Source/App/EbAppConfig.c

```diff
@@ -324,7 +332,13 @@ config_entry_t config_entry[] = {
 
     // Quantization
     { SINGLE_INPUT, QP_TOKEN, "QP", SetCfgQp },
+    { SINGLE_INPUT, USE_QP_FILE_TOKEN, "UseQpFile", SetCfgUseQpFile },
```

Comment:
UseQpFile is duplicated. Pls clean up.

Reply:
Removed duplicate code and pushed code (commit:67f68d8)